caikit_nlp/modules/token_classification/filtered_span_classification.py

```diff
@@ -115,7 +115,7 @@ def run(
         span_list = self.span_splitter.run(text)
         # Run each span through the classifier and determine based
         # on threshold and labels_to_output what results should be returned
-        text_list = [span.text for span in span_list]
+        text_list = [span.text for span in span_list.results]
```

Comment:
should probably also update the docstring parts that say
```
Span splitter that returns List[Span]
```
and since a task is added, we can do a quick check on init that the `span_splitter` module has the expected `TokenizationTask`?